scripts/mkcloudhost/allocpool

```diff
@@ -19,6 +20,19 @@ sleep(rand(2)); # reduce chance of collisions with make -j
 # lock+select testdir
 my $found=0;
 foreach my $d (@dirs) {
+        # check that allocations isn't out of sync with pool
+        open my $fha, $allocationsfile or die "Error: cant read $allocationsfile";
+        my $allocationfound=0;
+        my $n=$d;
+        $n=~ s/$pooldir\///;
+        while(<$fha>) {
+                if(/^[a-z]*$n jenkins$/) {
```

Comment:
nit: you could use an explicit loop variable here for readability. I suspect that most people who don't deal with Perl on a regular basis might not be familiar with the subtleties of `$_` :-)

Reply:
Done, please re-review